source/features/recently-pushed-branches-enhancements.tsx

```diff
@@ -13,30 +13,30 @@ import {getRepoURL} from '../libs/utils';
 
 const repoUrl = getRepoURL();
 
-async function init() {
+function getList() {
 	const fragmentURL = `/${repoUrl}/show_partial?partial=tree%2Frecently_touched_branches_list`;
-	if (select.exists(`[data-url='${fragmentURL}'], [src='${fragmentURL}']`)) {
-		return false;
-	}
-
-	const codeTabURL = select<HTMLAnchorElement>('[data-hotkey="g c"]').href;
-	const response = await fetch(codeTabURL);
-	const html = await response.text();
-
-	// https://github.com/sindresorhus/refined-github/issues/216
-	if (html.includes(fragmentURL)) {
-		select('.repository-content').prepend(<include-fragment src={fragmentURL}></include-fragment>);
-	}
+
+	return (
+		select(`[data-url='${fragmentURL}'], [src='${fragmentURL}']`) || // Already on page
```

Comment:
Explicit `if-else` using `select.exists()` would be a better choice here.

Took me a minute to understand what was happening, with all the combined logical OR and inline comments.

Reply:
Indeed. Should be better now